### ISSUE TITLE
Adding a world to your server from MH Marketplace wipes all skripts and opped players

### ISSUE THREAD

**Issue by Centrect** [Minehut/Meta#613]
### Bug report

**Checklist**
- [X] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [X] I am NOT reporting a security vulnerability
- [X] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [X] This issue does not contain any sensitive information

**Platform**

Both

**Describe the bug**

When adding a world from the Minehut Marketplace, on joining the game, it wipes all skript files, playerdata files etc.

**To reproduce**
Steps to reproduce the behaviour:
1. Add a Minehut Marketplace world to your server
2. Join the server
3. Observe results

**Expected behaviour**

When a world is simply added, it should NOT wipe all data from your server.

**Device Information**

- Minecraft client (and version): Lunar Client 1.16.5
- Web browser (and version): Google Chrome 90.0.4430.93 (Official Build) (64-bit)
- Operating system (and version): Windows 10
- Minehut server name: Eravate

**Screenshots**

Not really applicable here

**Additional context**

---
**Try to reproduce this bug and let us know in the comments if you can reproduce it or not. Make sure to include your relevant device information!**

**Comment by drtshock** [comment]
Which content pack did this? This might be something that's an issue with that specific pack. Please reach out to support if this happens again so we can resolve this for you.

On a more generic note, should we update the install process at all to make anything more clear?

**Comment by ahoulton** [comment]
> 
> 
> Which content pack did this? This might be something that's an issue with that specific pack. Please reach out to support if this happens again so we can resolve this for you.
> 
> On a more generic note, should we update the install process at all to make anything more clear?

I installed a skript from market on my dev server a few days ago and all the folders in plugins/Skript/scripts were deleted when I restarted it, not sure if this is known already, but it's worth pointing out.

Generally, the installation process for market content should be a bit more clearer. Maybe a "more information" when you go to install the market content what folders/files will be replaced/deleted and maybe what folders/files will be added.

**Comment by Centrect** [comment]
I agree, and it wasnt just a world, it was skript files too. Packs that I can remember were the Easy Punish skript, Castle Hub, Twitch Rivals Bowbash and Simplepolls. I'm almost certain that this is a reproducable issue rather than a one off bug.

**Comment by KingPr0o7** [comment]
I have a similar issue with this but it was a instead of a world I just used the upload feature with skripts. When I restart the server the server goes down for like 4.5sec and then boots back up with all skripts not working. Even though they are still there.

**Comment by orvit-png** [comment]
If this is an actual issue, make sure that there is at least a warning that this may happen, or an option to download them as a zip/bz file.
